unattended_installer/install_functions/installMain.sh

```diff
@@ -269,12 +275,6 @@ function main() {
         checks_names
```

Comment:
This function must be executed before installing anything.

Reply:
This function needs the wazuh-install-files.tar, which is not created yet